### PR TITLE
Move issue template to yml; add form.

### PR DESCRIPTION
**Pull request by roidelapluie**
Example in https://github.com/roidelapluie/prometheus/issues/new?assignees=&labels=&template=bug_report.yml

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+description: Create a report to help us improve.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for opening a bug report towards Prometheus.
```
Comment (LeviHarrison) [inline .github/ISSUE_TEMPLATE/bug_report.yml:8]: nit: word choice

```suggestion
        Thank you for opening a bug report for Prometheus.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      description: Insert relevant configuration here. Don't forget to remove secrets.
+      render: yaml
+  - type: textarea
+    attributes:
+      label: Alertmanager version
+      description: Insert output of `alertmanager --version` here (if relevant to the issue).
+  - type: textarea
+    attributes:
+      label: Alertmanager configuration file
+      description: Insert relevant configuration here (if relevant to the issue). Don't forget to remove secrets.
```
Comment (LeviHarrison) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]: nit: unneeded repetition

```suggestion
      description: Insert relevant configuration here. Don't forget to remove secrets.
```

**Review by LeviHarrison — COMMENTED** [review]
Thanks! This is a nice improvement to the bug reporting process. Just a few nits.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      value: |
+        Thank you for opening a bug report towards Prometheus.
+
+        Please do *NOT* ask support questions in Github issues.
+
+        If your issue is not a feature request or bug report use our [community support](https://prometheus.io/community/).
+
+        There is also [commercial support](https://prometheus.io/support-training/) available.
+  - type: textarea
+    attributes:
```
Comment (Nexucis) [inline .github/ISSUE_TEMPLATE/bug_report.yml:16]: I would add that : 

```yaml
    validations:
      required: true
```

Comment (Nexucis) [reply .github/ISSUE_TEMPLATE/bug_report.yml:16]: and same for `What did you expect to see?` and `What did you see instead? Under which circumstances?` sections

**Review by Nexucis — APPROVED** [review]
lgtm, thanks!

**Pull request merged**